examples/mongodbatlas_advanced_cluster/global-cluster/main.tf

```diff
@@ -54,17 +103,16 @@ resource "mongodbatlas_advanced_cluster" "cluster" {
         instance_size = "M10"
         node_count    = 1
       }
-      provider_name = "GCP"
-      priority      = 0
-      region_name   = "US_EAST_4"
+      provider_name = "AZURE"
```

Comment:
curious why we're changing the provider in this example

Reply:
We're not changing exactly, the prior example used all 3 providers, now we are using only two mostly just to reduce the verbosity of the example as now we have to specify several replication_specs for each shard